crates/fluvio-version-manager/src/command/itself/install.rs

```diff
@@ -25,19 +25,9 @@ pub struct SelfInstallOpt;
 
 impl SelfInstallOpt {
     pub async fn process(&self, notify: Notify) -> Result<()> {
-        // Checks if FVM is already installed
-        let bin_path = fvm_bin_path()?;
-
-        if bin_path.exists() {
-            notify.info(format!(
-                "FVM is already installed at {}",
-                bin_path.display()
-            ));
-            return Ok(());
-        }
-
         let fvm_installation_path = self.install_fvm()?;
-        Settings::init()?;
+
+        Settings::open()?;
```

Comment:
`Settings::init` cretes the `Settings` file if it doesnt exists, instead `Settings::open` will attempt to open the `settings.toml` file, but if it doesnt
exists it creates it (using `Settings::init`).